thorlcr/activities/funnel/thfunnelslave.cpp

```diff
@@ -682,6 +680,8 @@ class NonEmptySlaveActivity : public CSlaveActivity
         sendReceiving = false;
         masterMpTag = TAG_NULL;
         appendOutputLinked(this);
+        if (container.queryLocalOrGrouped())
```

Comment:
why not true for global?  I don't think it can ever be dependent on onStart()?  Does it need to be true if it requires global coordination?